sequencing/ATACSeq/Rscripts/collateSexChecks.r

```diff
@@ -1,34 +1,65 @@
-#
+## ==========================================================================================##
+##                      ATAC-seq pipeline STEP 5.3: Collate sex checks                       ##
+## ==========================================================================================##
+## EXECUTION: Rscript ./Rscripts/collateSexChecks.r <dataPath>                               ||
+## - execute from scripts directory                                                          ||
+##                                                                                           ||
+## DESCRIPTION: This scripts collates results from calling peaks in sex chromosomes and      ||
+## uses this to check the assigned sex of samples.                                           ||
+##                                                                                           ||
+## INPUTS:                                                                                   ||
+## - <config file> : R config file for project                                               ||
+##                                                                                           ||
+## OUTPUTS:                                                                                  ||
+## - sexPredictions.csv, sexPredictions.pdf                                                  ||
+##                                                                                           ||
+## REQUIRES:                                                                                 ||
+## - R/4.2.1-foss-2022a, libraries: ggpubr, tidyverse, conflicted,ggplot2                    ||
+## ==========================================================================================##
+
 args<-commandArgs(trailingOnly=TRUE)
-dataPath<-args[1]
+configFile<-args[1]
+source(configFile)
 
 library(ggplot2)
 library(ggpubr)
 library(tidyverse)
-
-
+library(conflicted)
+conflict_prefer("filter", "dplyr")
+conflict_prefer("lag", "dplyr")
```

Comment:
We could probably avoid this extra dependency in `conflicted` if namespaces are used.

So you could do:
```R
dplyr::filter(...)
dplyr::lag(...)
```

Reply:
Also, I don't even see you using `filter` or `lag`, so why use `conflicted` at all?

Reply:
I am guessing it is just to avoid the conflict message to pop up

Reply:
Nah, if anything it makes more logs than normal. Here's the output:

### test.R
```R
library(dplyr)
library(conflicted)
conflict_prefer("filter", "dplyr")
conflict_prefer("lag", "dplyr")
```

### output
```bash
$ Rscript test.R

  Attaching package: ‘dplyr’
  
  The following objects are masked from ‘package:stats’:
  
      filter, lag
  
  The following objects are masked from ‘package:base’:
  
      intersect, setdiff, setequal, union
  
  [conflicted] Will prefer dplyr::filter over any other package.
  [conflicted] Will prefer dplyr::lag over any other package.
```

Reply:
I had several issues with this script, so was probably being used before and I got rid of those functions